src/goLanguageServer.ts

```diff
@@ -421,29 +422,25 @@ function registerUsualProviders(ctx: vscode.ExtensionContext) {
 	vscode.workspace.onDidChangeTextDocument(parseLiveFile, null, ctx.subscriptions);
 }
 
+const acceptGoplsPrerelease = false;
```

Comment:
Am guessing you want to expose this via a setting? Have we decided if this will go inside the `gopls` setting or will we add an official setting?

Reply:
This is actually a switch for the Go Nightly extension - one of the patches converts the value to true for Go Nightly, false for the regular Go extension. It's not something that could be a `gopls` setting because it's about which version of `gopls` to install rather than something `gopls` can use.

Reply:
got it